rust/operator-binary/src/crd/authentication.rs

```diff
@@ -34,35 +31,33 @@ pub enum Error {
     ))]
     AuthenticationClassProviderNotSupported {
         authentication_class_provider: String,
-        authentication_class: ObjectRef<AuthenticationClass>,
+        authentication_class: ObjectRef<auth_core::v1alpha1::AuthenticationClass>,
     },
 
     #[snafu(display(
         "Nifi doesn't support skipping the LDAP TLS verification of the AuthenticationClass {authentication_class}"
     ))]
     NoLdapTlsVerificationNotSupported {
-        authentication_class: ObjectRef<AuthenticationClass>,
+        authentication_class: ObjectRef<auth_core::v1alpha1::AuthenticationClass>,
     },
 
     #[snafu(display("invalid OIDC configuration"))]
-    OidcConfigurationInvalid {
-        source: stackable_operator::commons::authentication::Error,
-    },
+    OidcConfigurationInvalid { source: auth_core::v1alpha1::Error },
```

Comment:
Typically we've been using the fully-qualified types for external error sources:

```suggestion
    OidcConfigurationInvalid {
        source: stackable_operator::crd::authentication::core::v1alpha1::Error,
    },
```

Reply:
a41b4497916ac5681281e1ef157d88de68f5f8ef